sdk/python/feast/data_source.py

```diff
@@ -565,6 +591,7 @@ def __init__(
         created_timestamp_column: Optional[str] = "",
         field_mapping: Optional[Dict[str, str]] = None,
         date_partition_column: Optional[str] = "",
+        s3_endpoint_override: Optional[str] = "",
```

Comment:
Is there a reason this doesn't have a default value of `None`?

Reply:
fixed